lib/common/asar.js

```diff
@@ -582,8 +581,7 @@
       };
       mkdirSync = fs.mkdirSync;
       fs.mkdirSync = function(p, mode) {
-        var filePath, isAsar, ref;
-        ref = splitPath(p), isAsar = ref[0], filePath = ref[2];
+        const [isAsar, , filePath] = splitPath(p);
```

Comment:
Yeah, I saw this in the mdn docs and was pretty impressed that it is supported.

Yeah, not sure about a repl, there must be one out there.